jenkins_integration/jenkins_integration.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+//Create curl post request to hit the Jenkins webhook
+$secrets = json_decode(file_get_contents($_SERVER['HOME'] . '/files/private/secrets.json'));
+
+$curl = curl_init($secrets->jenkins_url);
+
+//Setup header with authentication
+curl_setopt($curl, CURLOPT_HTTPHEADER, array('Content-Type: application/json',
+    'Authorization: Basic '. base64_encode("$secrets->username:$secrets->api_token"),
+));
+
+//Declare request as a post and setup the fields
+curl_setopt($curl, CURLOPT_POST, true);
+curl_setopt($curl, CURLOPT_POSTFIELDS, array(
+  'token' => $secrets->token,
+));
+
+//Execute the request (could store response if necessary)
+curl_exec($curl);
```

Comment:
Riiiiight... Thanks a _lot_, Jenkins. 

IIRC it might not be hard to script an API-call two-step to allow this, but could also be a little too project specific since I guess you'd have to have some other attribute to match on...